modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/A11yTool.path

```diff
@@ -12,7 +12,17 @@
 <tbody>
 <tr>
 	<td>ACCESSIBILITY_PANEL_HEADING</td>
-	<td>//span[contains(text(),'Accessibility Violations')]</td>
+	<td>//div[contains(@class,'a11y-panel--violations-header')]//span[contains(text(),'Accessibility Violations')]</td>
```

Comment:
typo? `a11y-panel-violations-header`

Reply:
cc/ @matuzalemsteles typo on the attribute?

Reply:
This is partially correct, it's a pattern called BEM that we're applying here or just borrowing, `a11y-panel` is the scope and `--violations-header` would be the modifier of the in-scope element and one way to separate this information is using two dashes, in this case I think instead of using the two dashes, we should be using two underscores because it is more related to the element than state change which is recommended to use two dashes.

Reply:
Thanks @matuzalemsteles , let me know how to track that underscores update (LPS ticket) so we can include any test fixes with it.